charts/nri-prometheus/README.md

```diff
@@ -1,6 +1,6 @@
 # nri-prometheus
 
-![Version: 2.1.8](https://img.shields.io/badge/Version-2.1.8-informational?style=flat-square) ![AppVersion: 2.16.2](https://img.shields.io/badge/AppVersion-2.16.2-informational?style=flat-square)
+![Version: 2.1.8](https://img.shields.io/badge/Version-2.1.8-informational?style=flat-square) ![AppVersion: 2.16.3](https://img.shields.io/badge/AppVersion-2.16.3-informational?style=flat-square)
```

Comment:
We are removing this in all PRs: https://github.com/newrelic/nri-prometheus/blob/b5e4a4d20afd1b571c2c7db6ad8c0a93cbae7e52/charts/nri-prometheus/README.md.gotmpl#L4

You can remove it in this PR if you wish.